frontend/src/pages/learn/index.tsx

```diff
@@ -20,6 +20,7 @@ import rulesOfThumbForPublicPensionsSmImage from '../../../public/assets/rules-o
 import { HeroBanner } from '../../components/HeroBanner'
 import Layout from '../../components/Layout'
 import { getDCTermsTitle } from '../../utils/seo-utils'
+import { useRouter } from 'next/router'
```

Comment:
`useRouter` is not used, should be removed

Reply:
Fixed